gbpservice/nfp/service_vendor_agents/vyos/vyos_session/utils.py

```diff
@@ -68,6 +69,13 @@ def init_logger(logger):
     # add the handlers to the logger
     logger.addHandler(file_handler)
 
+    formatter = logging.Formatter('vyos %(name)s %(funcName)s() %(levelname)s '
+                                  '%(message)s')
```

Comment:
Is this done to make the default log forwarding enabled to local host irrespective of log forwarding configured or not?